samples/spm/boards/nrf5340pdk_nrf5340_cpuapp.overlay

```diff
@@ -0,0 +1,24 @@
+/* SPDX-License-Identifier: LicenseRef-Nordic-5-Clause */
```

Comment:
I think this can be removed now since PDK support was removed just prior to 1.5.0